cft/deploy.yml

```diff
@@ -0,0 +1,38 @@
+Resources:
+  ElasticsearchDomain:
+    Type: AWS::Elasticsearch::Domain
+    Properties:
+      DomainName: "test"
+      ElasticsearchClusterConfig:
+        DedicatedMasterEnabled: true
+        InstanceCount: 2
+        ZoneAwarenessEnabled: true
+        InstanceType: "m3.medium.elasticsearch"
+        DedicatedMasterType: "m3.medium.elasticsearch"
+        DedicatedMasterCount: 3
+      EncryptionAtRestOptions:
+        Enabled: false
+      EBSOptions:
+        EBSEnabled: true
+        Iops: 0
+        VolumeSize: 20
+        VolumeType: "gp2"
+      SnapshotOptions:
+        AutomatedSnapshotStartHour: 0
```

Comment:
Coverity Rapid Scan found issue: Insecure TLS version - CWE-327, Low Severity

**The Amazon Elasticsearch Service (Amazon ES) domain supports TLS 1.0.
**

**How to fix:** Configure the ElasticSearch service domain to require TLS 1.2 or newer by explicitly setting the `Properties.DomainEndpointOptions.TLSSecurityPolicy` value to `Policy-Min-TLS-1-2-2019-07`.